lib/logger.js

```diff
@@ -305,7 +306,12 @@ class Logger {
             const msg = this._createMessage(info);
 
             // Also pass in default parameters
-            info = Object.assign({}, info, this.args);
+            if (info instanceof Error) {
+                // We want to preserve Error type so that standard banyan serializers kick in.
```

Comment:
*We want to preserve the Error type before bunyan serialisation kicks in*

Reply:
Done.